src/extensions/default/InlinePaddingEditor/InlinePaddingEditor.js

```diff
@@ -42,12 +42,13 @@ define(function (require, exports, module) {
      * @param {!string} padding  Initially selected padding
      * @param {!CodeMirror.TextMarker} marker
      */
-    function InlinePaddingEditor(padding, marker) {
+    function InlinePaddingEditor(padding, marker, type) {
```

Comment:
If we're going to share code between padding/margin editors, let's not use the same name in the files.  This is `InlinePaddingEditor` but you're referencing margin below, and switching on `type`.  We should probably find a better name, so it's not confusing?

Reply:
Ya I have written this in the description itself that I'll change the naming. I just wanted to show and discuss the code.
@humphd Should I replace `padding` to `padding-margin` everywhere ?

Reply:
Probably, yes.  Should we just call it `InlineBoxModelEditor` since that's basically what this is now?

Reply:
You mean `padding` should be replaced with `BoxModel` everywhere (i.e in paddingeditor.js, template.html, variables etc.) , right ?

Reply:
That's my suggestion, yes.

Reply:
Okay got it !